common/djangoapps/philu_commands/management/commands/autoscore_ondemand_course_ora.py

```diff
@@ -20,11 +20,24 @@ class Command(BaseCommand):
     help = """
     Auto score ORA assessment of on demand course, if learner has submitted ORA a certain number of days ago. Number
     of days are configurable from site configurations model though its default value is 3 days.
+
+    In-order to access site configurations, site id of LMS site is required as a command param, but if it is not
+    provided or its value is wrong then default ORA assessment waiting value will be 3 days.
```

Comment:
I'd prefer mentioning `DAYS_TO_WAIT_AUTO_ASSESSMENT` instead of 3.

Reply:
done